### PR TITLE
Make Coronastatusnp.com 

### PR DESCRIPTION
**Pull request by kalapathar**
Could you make a website for Nepal too - similar to Coronastatus.no.

### CODE REVIEW

File: app/locales/ne.json [hunk 1]
```diff
@@ -0,0 +1,200 @@
+{
```
Comment (fossecode) [inline app/locales/ne.json:1]: Is this file supposed to be named `ne.json` or should it be `np.json`?

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (fossecode) [inline .vscode/settings.json]: Could you add `.vscode/` to .gitignore? And remove this file :)

**Review by fossecode — CHANGES_REQUESTED** [review]
You also need to add a mapping to a flag :) https://github.com/BustByte/coronastatus/blob/master/app/domain/flags.ts

The flag key is `np`: https://flagicons.lipis.dev/

File: app/locales/ne.json (hunk 1 above)
Comment (kalapathar) [reply app/locales/ne.json:1]: `np` is not in this list: https://github.com/ladjs/i18n-locales so I think it should be `ne.json`

Comment (fossecode) [reply app/locales/ne.json:1]: Ah sorry, now I remember :)

File: .vscode/settings.json (hunk 2 above)
Comment (fossecode) [reply .vscode/settings.json]: https://stackoverflow.com/questions/1274057/how-to-make-git-forget-about-a-file-that-was-tracked-but-is-now-in-gitignore

**Comment by fossecode** [comment]
Could you also delete `package-lock.json`? 😅 we use `yarn` instead

**Comment by kalapathar** [comment]
> Could you also delete `package-lock.json`? 😅 we use `yarn` instead

Deleted `package-lock.json`

**Review by fossecode — APPROVED** [review]

**Pull request merged**